pkgs/tools/networking/ntp/default.nix

```diff
@@ -15,6 +15,10 @@ stdenv.mkDerivation rec {
     sha256 = "0whbyf82lrczbri4adbsa4hg1ppfa6c7qcj7nhjwdfp1g1vjh95p";
   };
 
+  # The hardcoded list of allowed system calls for seccomp is
+  # insufficient for NixOS, add more to make it work (issue #21136).
+  patches = [ ./seccomp.patch ];
```

Comment:
Can you also send this commit upstream?

Reply:
it also fails due to the use of the `openat` syscall when it tries to update the drift file

Reply:
I still think a blacklist is better. There is a whole class of *at system calls that might be used instead of their pendants by libc.